ViSearchSDK/ViSearchSDK/Classes/ViProductSearch.swift

```diff
@@ -24,11 +24,6 @@ open class ViProductSearch : NSObject {
     
     private var client : ViProductSearchClient? = nil
     
-    /// Default Constructor
-    private override init() {
```

Comment:
why do this? the setup() method is the same as init. app key and placement ID are mandatory.

Reply:
nvm get what u tried to do